packages/esm-patient-list-app/src/api/api.ts

```diff
@@ -0,0 +1,84 @@
+import { isOfflineUuid } from '@openmrs/esm-framework';
+import {
```

Comment:
Same as the comment above; avoid imports from adjacent index modules.

Reply:
Done!